calibration/CTDMOG/CGINS-CTDMOG-13419__20200222.csv

```diff
@@ -0,0 +1,25 @@
+serial,name,value,notes
+37-13419,CC_a0,-1.329466e-04,source_file 3305-00101-00793-A_SN_37-13419_QCT_Results_CTDMO-G.txt
+37-13419,CC_a1,3.135342e-04,
+37-13419,CC_a2,-4.996639e-06,
+37-13419,CC_a3,2.155307e-07,
+37-13419,CC_cpcor,-9.570000e-08,
+37-13419,CC_ctcor,3.250000e-06,
+37-13419,CC_g,-9.816976e-01,
+37-13419,CC_h,1.361321e-01,
+37-13419,CC_i,-1.352183e-04,
+37-13419,CC_j,2.956730e-05,
+37-13419,CC_p_range,1450,
+37-13419,CC_pa0,4.346035e-01,
+37-13419,CC_pa1,4.525311e-03,
+37-13419,CC_pa2,-2.832507e-11,
+37-13419,CC_ptca0,5.253855e+05,
+37-13419,CC_ptca1,9.342711e-01,
+37-13419,CC_ptca2,1.035358e-01,
+37-13419,CC_ptcb0,2.535188e+01,
+37-13419,CC_ptcb1,1.750000e-04,
+37-13419,CC_ptcb2,0.000000e+00,
+37-13419,CC_ptempa0,1.526920e+02,
+37-13419,CC_ptempa1,-7.077361e-02,
+37-13419,CC_ptempa2,7.240583e-07,
+37-13419,CC_wbotc,7.443328e-07,
```

Comment:
Looks good.

Reply:
good - this one is an instrument cal that was already pushed on the cals branch and got lumped in with Collin's glider cals.  Technically Allen created it during QCT so I approve!